src/restconfig/src/main/java/org/geoserver/rest/catalog/LayerGroupController.java

```diff
@@ -198,6 +203,11 @@ public boolean supports(MethodParameter methodParameter, Type targetType, Class<
 
     @Override
     public void configurePersister(XStreamPersister persister, XStreamMessageConverter converter) {
+        // add keyword object configuration
+        XStream xstream = persister.getXStream();
```

Comment:
Suspicious, why doesn't ServiceInfo and ResourceInfo need the same treatment? They also have a keywords field. Also  check XStreamPersister, a custom converter is registered in that class for ResourceInfo and ServiceInfo (even with the same output, it would be best to maintain consistency)